dcs-packaging-tool/dcs-packaging-tool-gui/src/main/resources/bundles/errors.properties

```diff
@@ -17,7 +17,7 @@ packagegeneration.packagecreation.error = Error creating package.
 packageparamscreation.error = Error Creating Package Generation Parameters.
 packagegeneration.save.error = Error Saving Package
 basedirectoryordescriptionnotselected.error = A base directory or an existing package description file must be selected to continue.
-packagenamemissing.error = A package name must be provided to continue.
+packagenamemissing.error = Please make sure to provide values for the "Required Fields" before continuing to the next step.

```

Comment:
In the interest of reserving some contracted hours for documentation work, this is something we can do on our side.